auto_submit/bin/server.dart

```diff
@@ -7,13 +7,17 @@ import 'dart:async';
 import 'package:appengine/appengine.dart';
 import 'package:auto_submit/helpers.dart';
 import 'package:auto_submit/requests/github_webhook.dart';
+import 'package:auto_submit/service/config.dart';
 import 'package:shelf_router/shelf_router.dart';
 
 Future<void> main() async {
   await withAppEngineServices(() async {
     useLoggingPackageAdaptor();
 
-    final Router router = Router()..post('/webhook', GithubWebhook().post);
+    Config config = Config();
+    GithubWebhook githubWebhook = GithubWebhook(config);
```

Comment:
final?

Reply:
Thanks Keyong, I will check for all variables and add final if necessary.